sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -373,6 +381,55 @@ def _copy_callback(raw_response, _, headers):  # pylint: disable=unused-argument
             **kwargs
         )
 
+    @distributed_trace
+    def begin_create_composed_model(
+        self,
+        model_ids,
+        **kwargs
+    ):
+        # type: (List[str], Any) -> LROPoller[CustomFormModel]
+        """Creates a composed model from a collection of existing trained models with labels.
+
+        :param list[str] model_ids: List of model IDs that were trained with labels.
```

Comment:
a composed model could contained composed models. I wonder if we should add that here too or just have something generic like
`List of model ids to use in the composed model.`